From 9160e86fe2a6324d7259db5959dbf366b500c1df Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Mon, 8 Jun 2020 06:39:07 +0530
Subject: [PATCH] gutils: GetModuleHandle is not available on WinRT

We're also using these DLLs for fetching user profile dirs which are
not accessible under WinRT anyway, so just ifdef that out.
---
 glib/gutils.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/glib/gutils.c b/glib/gutils.c
index 81eea9f..5835bed 100644
--- a/glib/gutils.c
+++ b/glib/gutils.c
@@ -1879,6 +1879,7 @@ g_get_user_config_dir (void)
 
   G_LOCK (g_utils_global);
 
+#ifndef G_WINAPI_ONLY_APP
   if (g_user_config_dir == NULL)
     g_user_config_dir = g_build_user_config_dir ();
   user_config_dir = g_user_config_dir;
@@ -1906,6 +1907,7 @@ g_build_user_cache_dir (void)
       cache_dir = g_build_filename (home_dir, ".cache", NULL);
       g_free (home_dir);
     }
+#endif
 
   return g_steal_pointer (&cache_dir);
 }
-- 
2.38.1

